configs/config.toml.SAMPLE

```diff
@@ -43,6 +43,17 @@
 #   # Uncomment to override the node's default addresses announced in the network
 #   # AnnouncedAddresses = ["/dns4/example.com/tcp/3919", "/ip4/80.70.60.50/tcp/3919"]
 
+[Client]
+# Look-back period to check if existing keeps are awaiting signer generation.
+# The value should be provided based on the sanctioned application requirements.
+#  AwaitingKeyGenerationLookback = "24h"	# optional
+
+# Timeouts for processes execution. Within these timeouts the process will keep
+# retrying to generate a signer or calculate a signature. The values should be
+# provided based on the sanctioned application requirements.
+#  KeyGenerationTimeout = "3h" 				# optional
```

Comment:
What's the reasoning behind separating `AwaitingKeyGenerationLookback` and `KeyGenerationTimeout`? There is no point in looking back further than `KeyGenerationTimeout`.

Reply:
It's used in two different ways. When starting a client we may want to look back for keeps from e.g. 2 days, but having such a long timeout for a single key generation process doesn't make much sense. It's strictly related to the application but having it separated makes the client more flexible and configurable.